x-pack/test/case_api_integration/security_and_spaces/tests/trial/cases/push_case.ts

```diff
@@ -78,70 +65,12 @@ export default ({ getService }: FtrProviderContext): void => {
       await actionsRemover.removeAll();
     });
 
-    const createCaseWithConnector = async ({
```

Comment:
This was moved to the utils file to make it shared between these tests and the ones in `spaces_only`.